detection-rules/impersonation_employee_subject.yml

```
@@ -0,0 +1,37 @@
+name: "BEC: Employee impersonation with subject manipulation"
+description: |
+    Subject matches the display name of someone in your organization, and the body resembles a BEC attack.
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+
+  // attacks have been observed where the impersonation is
+  // in the subject, and the display name contains what a
+  // subject typically would contain. this is an attempt to
+  // bypass impersonation detections
+  and subject.subject in~ $org_display_names
```

```suggestion
  and any($fake_org_display_names, strings.contains(subject.subject, .))
```
Do we want this instead? So instead of subject:"sam scholten" it would also match "urgent click here sam scholten"